src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -61,4 +82,26 @@ public CreateCaseCallbackService(
             .filter(item -> !Strings.isNullOrEmpty(item.getTransformationUrl()))
             .getOrElse(Validation.invalid("Transformation URL is not configured"));
     }
+
+    private Validation<Seq<String>, Map<String, Object>> createNewCase(
+        ExceptionRecord exceptionRecord,
+        ServiceConfigItem configItem,
+        long caseId
+    ) {
+        try {
+            log.info("Start creating exception record for {} {}", configItem.getService(), caseId);
+
+            transformationClient.transformExceptionRecord(
+                configItem.getTransformationUrl(),
+                exceptionRecord,
+                "s2s token"
+            );
+
+            return Validation.valid(ImmutableMap.of("caseReference", UUID.randomUUID()));
+        } catch (Exception exception) {
+            log.error("Failed to create exception for {} {}", configItem.getService(), caseId, exception);
+
+            return Validation.invalid(Array.of("Internal error. " + exception.getMessage()));
```

Comment:
There are two possible error scenarios:
- 422 from transformation API - we should return a 200 response with those errors, so that they can be displayed to the caseworker
- any other error (5xx, timeouts, our internal errors), including 400 - the orchestrator should return a 500 response and CCD will display a generic error

Reply:
Good catch. Transformation client already handles 422 and 400 altogether. Just about to extract to separate catch block and we can proceed from there as I'm not entirely sure whether re-throw or return some internal. But that would be similar to `Exception` handle